app.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
I think this file is needed in order to have the ability to create an ephemeral Heroku application to run tests against. I'm not confident that is the case but I am reading into this case.